src/site/_includes/partials/subscribe.njk

```diff
@@ -0,0 +1,66 @@
+<web-subscribe id="subscribe">
+  <div class="w-subscribe">
+    <div>
+      <h1>Become a better web developer.</h1>
+      <p id="w-subscribe__message">Stay up to date with the latest about web platform.</p>
```

Comment:
We have some CSS that we use to create spacing around elements with IDs. This is so when someone deep links to an id, the header doesn't overlap the content. In this case it's making additional space around the `p` element. So if you don't need the `id` it might be good to remove it to keep things tidy.

Reply:
The ids were used for query selectors in the Subscribe custom element, do you have any recommendations as to how I should do that? (By tag or class?)

Reply:
I'd suggest doing it by class.